### PR TITLE
Adds a new crate to be ordered from cargo

### PR DESCRIPTION
**Pull request by jughu**
For all your cheffing needs!
Adds a crate that be can ordered from cargo with condimemts for 20 points it includes
Coldsauce, Hotsauce, Soysauce, taste enchancer, ketchup, salt, pepper,

🆑 Improvedname
rscadd: Adds a crate with kitchen condiments to cargo
/🆑

### CODE REVIEW

**Comment by Plotron** [comment]
Why 20 points? Should be 15 or even 10.

**Comment by jughu** [comment]
BECAUSE LIFE AINT CHEAP KIDDO

**Comment by jughu** [comment]
Plus frostoil can used in a antag thing like changelings

**Comment by Plotron** [comment]
Coldsauce = frostoil? 

Okay...

File: code/datums/supplypacks.dm [hunk 1]
```diff
@@ -885,6 +885,20 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
 	cost = 60
 	containername = "Pizza crate"
 
+/datum/supply_packs/organic/condiments
+        name = "Condiments crate"
+        contains = list(/obj/item/weapon/reagent_containers/food/condiment/coldsauce,
+                                        /obj/item/weapon/reagent_containers/food/condiment/hotsauce,
+                                        /obj/item/weapon/reagent_containers/food/condiment/lifeenchancer,
```
Comment (FalseIncarnate) [inline code/datums/supplypacks.dm]: The bottle you added is tasteenhancer not lifeenhancer.

Typo?

**Comment by jughu** [comment]
Yes coldsauce is basically frostoil and lings cyrosting is frostoil too and throw that in a sleepy pen and you can fake lings and hotsauce is capsaucin put some other chems with it and you have pepperspray

**Comment by jughu** [comment]
O i knew i typod thanks man

File: code/datums/supplypacks.dm [hunk 2]
```diff
@@ -885,6 +885,20 @@ var/list/all_supply_groups = list(supply_emergency,supply_security,supply_engine
...
+        name = "Condiments crate"
+        contains = list(/obj/item/weapon/reagent_containers/food/condiment/coldsauce,
+                                        /obj/item/weapon/reagent_containers/food/condiment/hotsauce,
+                                        /obj/item/weapon/reagent_containers/food/condiment/tasteenchancer,
+                                        /obj/item/weapon/reagent_containers/food/condiment/sugar,
+                                        /obj/item/weapon/reagent_containers/food/condiment/soysauce,
+                                        /obj/item/weapon/reagent_containers/food/condiment/ketchup,
+                                        /obj/item/weapon/reagent_containers/food/condiment/peppermill,
+                                        /obj/item/weapon/reagent_containers/food/condiment/saltshaker)
+       cost = 20
```
Comment (FalseIncarnate) [inline code/datums/supplypacks.dm]: Compile error says you've got inconsistent indentation somewhere around this line. Double-check to make sure you didn't accidentally throw a space somewhere in the tabbing. This should all be coming up on your Dreammaker output when you compile your changes in it. If you aren't making the changes in DreamMaker or are not compiling them, please start doing so.

**Comment by jughu** [comment]
I do it later and i just did it off github editor currently

**Comment by Fox-McCloud** [comment]
>I do it later and i just did it off github editor currently

We're not going to accept a PR done in the Github editor.

Do it the right way.

**Comment by jughu** [comment]
Oke i will make a new PR then done in the dreammaker

**Pull request closed without merging**